website/templates/emails/welcome_osf4i.html.mako

```diff
@@ -0,0 +1,47 @@
+<%inherit file="notify_base.mako" />
+
+<%def name="content()">
+<tr>
+  <td style="border-collapse: collapse;">
+    <h3 class="text-center" style="padding: 0;margin: 0;border: none;list-style: none;font-weight: 300;text-align: center;">Welcome to the OSF!</h3>
+  </td>
+</tr>
+<tr>
+  <td style="border-collapse: collapse;">
+
+Hello ${user.fullname},<br>
+<br>
+Thank you for verifying your OSF account, a free, open-source service maintained by the Center for Open Science. Here are a few things you can do with the OSF:<br>
```

Comment:
this is copied from the regular welcome email. Should that one be updated as well?